infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -210,28 +210,9 @@ public static void main(String[] args) {
                         opts.get("--log-path")).reset());
 
         // Pick the correct logging level before outputting error messages.
-        Logger root = (Logger) LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME);
-        switch ((String) opts.get("--log-level")) {
-            case "ERROR":
-                root.setLevel(Level.ERROR);
-                break;
-            case "WARN":
-                root.setLevel(Level.WARN);
-                break;
-            case "INFO":
-                root.setLevel(Level.INFO);
-                break;
-            case "DEBUG":
-                root.setLevel(Level.DEBUG);
-                break;
-            case "TRACE":
-                root.setLevel(Level.TRACE);
-                break;
-            default:
-                root.setLevel(Level.INFO);
-                log.warn("Level {} not recognized, defaulting to level INFO",
-                        opts.get("--log-level"));
-        }
+        final Logger root = (Logger) LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME);
+        final Level level = Level.toLevel(((String)opts.get("--log-level")).toUpperCase());
```

Comment:
What happens when the option is not there. The Docopts thing sets it to INFO ?

Reply:
Yes, same behavior as before